HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 History
 -------
 
+v0.14.1 (2018-10-27)
```

Comment:
tiny thing, could you put `v0.14.1 (2018-XX-XX)` to indicate this patch hasn't been released.

Reply:
Ok